site/app/controllers/pdf/PDFController.php

```diff
@@ -199,10 +249,10 @@ public function showGraderPDFEmbedded(string $gradeable_id) {
         $is_anon = $_POST['is_anon'] ?? false;
```

Comment:
```suggestion
        $is_anon = (bool) $_POST['is_anon'] ?? false;
```
Let's do the conversion once instead of everywhere it's used (this applies to `showGraderImageEmbedded` as well)